frontend/src/pages/connectionTypes/manage/advanced/NumericAdvancedPropertiesForm.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import {
+  ExpandableSection,
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { NumericField } from '~/concepts/connectionTypes/types';
+import { AdvancedFieldProps } from '~/pages/connectionTypes/manage/advanced/types';
+import NumberInputWrapper from '~/components/NumberInputWrapper';
+import TypeaheadSelect from '~/components/TypeaheadSelect';
+
+const StandardUnits = ['MiB', 'GiB', 'Hours', 'Minutes', 'Seconds'];
+
+enum Settings {
+  Min,
+  Max,
+}
+
+const NumericAdvancedPropertiesForm: React.FC<AdvancedFieldProps<NumericField>> = ({
+  properties,
+  onChange,
+  onValidate,
+}) => {
+  const [expanded, setExpanded] = React.useState<boolean>(false);
+  const [lastTouched, setLastTouched] = React.useState<Settings>(Settings.Max);
+
+  const isValid = React.useMemo(
+    () =>
+      (properties.min ?? Number.NEGATIVE_INFINITY) < (properties.max ?? Number.POSITIVE_INFINITY),
+    [properties.max, properties.min],
+  );
+  React.useEffect(
+    () => onValidate(isValid),
+    // do not run when callback changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [isValid],
+  );
+
+  const standardUnitOptions = React.useMemo(() => {
+    const options = StandardUnits.map((unit) => ({ value: unit, content: unit }));
+    if (properties.unit && !StandardUnits.includes(properties.unit)) {
+      options.push({ value: properties.unit, content: properties.unit });
+    }
+    return options;
+  }, [properties.unit]);
+
+  return (
+    <ExpandableSection
+      toggleText="Advanced settings"
+      onToggle={(_ev, isExpanded) => setExpanded(isExpanded)}
+      isExpanded={expanded}
+      data-testid="advanced-settings-toggle"
+    >
+      <FormGroup className="pf-v5-u-ml-lg" label="Unit" fieldId="unit-select">
```

Comment:
Use `FormFieldGroupExpandable` ?

Reply:
Created the `ExpandableFormSection` common component